pkg/server/admin.go

```diff
@@ -962,9 +963,22 @@ func (s *adminServer) Cluster(
 	if clusterID == (uuid.UUID{}) {
 		return nil, grpc.Errorf(codes.Unavailable, "cluster ID not yet available")
 	}
+
+	// Check if enterprise features are enabled.  We currently test for the
```

Comment:
maybe something like https://github.com/cockroachdb/cockroach/commit/64fd4d3aacc102cb1021bfac4100c2cefb4ff23c

Reply:
another option, that I like slightly more but don't feel that strongly about, would be to have some string->string `analyticsTags` or such that isn't "enterprise" or "license" specific, with a ccl hook that populates a "enterprise" tag, to avoid "enterprise" handling in the pure FLOSS code. That said, our commitment thus far has just been that we will ensure that it is possible to build a pure FLOSS binary, and there's already places where it shows that it was extracted from the full CCL codebase (e.g. the enterprise statements have to live in the FLOSS grammar and parser ASTS, even their execution isn't implemented), so I don't think it matters that much.